recipes/install.rb

```diff
@@ -26,6 +26,7 @@
 when 'mariadb'
   Array(node['mysqld']['mariadb']['packages']).each { |pkg| package pkg }
 when 'percona'
+  include_recipe 'mysqld::percona_repository'
```

Comment:
A comment explaining why the repository is set up would be nice for the future. Especially since we're not doing it for mariadb.